utils/pip-allow-failures.sh

```
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+PIP_COMMAND=${PIP_COMMAND:-pip}
+PIP_OPTIONS=${PIP_OPTIONS:-}
+REQUIREMENTS_FILE=$1
+
+while read dependency; do
+    dependency_stripped="$(echo "${dependency}" | sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//')"
+    # Skip comments
+    if [[ $dependency_stripped == \#* ]]; then
+        continue
+    # Skip blank lines
+    elif [ -z "$dependency_stripped" ]; then
+        continue
+    else
+        if $PIP_COMMAND install $PIP_OPTIONS "$dependency_stripped"; then
```

I find it useful to not swallow the errors if the user still need to fix and install this or that dependency. It's just a wrapper, if someone wants to ignore the output he should do it.